cni/network/invoker_cns.go

```diff
@@ -166,8 +169,8 @@ func (invoker *CNSIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, erro
 		// Do we want to leverage this lint skip in other places of our code?
 		key := invoker.getInterfaceInfoKey(info.nicType, info.macAddress)
 		switch info.nicType {
-		case cns.DelegatedVMNIC:
-			// only handling single v4 PodIPInfo for Frontend NICs at the moment, will have to update once v6 gets added
+		case cns.DelegatedVMNIC, cns.BackendNIC:
```

Comment:
i would say create separate case for  `cns.BackendNIC` and define new function `addBackendNictoResult` instead of changing `configureSecondaryAddResult`. we should return error if pnp id is missing in cns goalstate

Reply:
I think CNS will catch this issue up before it sends pnp id to CNI if this id is missing?